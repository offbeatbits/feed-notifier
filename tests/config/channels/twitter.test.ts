import dotenv from 'dotenv';
import path from 'path';

describe('Twitter config', () => {
  dotenv.config({
    path: path.resolve(process.cwd(), '.env.testing'),
  });

  it('should export Twitter config', () => {
    expect(require('../../../src/config/channels/twitter')).toMatchSnapshot(); // eslint-disable-line
  });
});
