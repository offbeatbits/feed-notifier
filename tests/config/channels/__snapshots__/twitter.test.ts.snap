// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Twitter config should export Twitter config 1`] = `
Object {
  "twitter": Object {
    "enabled": true,
    "username": "test_account",
  },
}
`;
